src/scribe_data/cli/main.py

```diff
@@ -179,22 +183,56 @@ def main() -> None:
         epilog=CLI_EPILOG,
         formatter_class=lambda prog: argparse.HelpFormatter(prog, max_help_position=60),
     )
-    convert_parser._actions[0].help = "Show this help message and exit."
+
+    # Setting up the arguments for the convert command
     convert_parser.add_argument(
-        "-f", "--file", type=str, help="The file to convert to a new type."
+        "-lan",
+        "--language",
+        type=str,
+        required=True,
+        help="The language of the file to convert.",
+    )
+    convert_parser.add_argument(
+        "-dt",
+        "--data-type",
+        type=str,
+        required=True,
+        help="The data type(s) of the file to convert (e.g., noun, verb).",
+    )
+    convert_parser.add_argument(
+        "-i",
+        "--input-file",
+        type=Path,
+        required=True,
+        help="The path to the input file to convert.",
     )
     convert_parser.add_argument(
         "-ot",
         "--output-type",
         type=str,
         choices=["json", "csv", "tsv", "sqlite"],
+        required=True,
         help="The output file type.",
     )
+    convert_parser.add_argument(
+        "-od",
+        "--output-dir",
+        type=Path,
+        required=True,
```

Comment:
There's no need for this to be required. Ideally the default directories would be used if this isn't included :)